pkg/scheduler/framework/plugins/volumebinding/binder.go

```diff
@@ -46,8 +46,8 @@ import (
 	"k8s.io/klog/v2"
 	v1helper "k8s.io/kubernetes/pkg/apis/core/v1/helper"
 	pvutil "k8s.io/kubernetes/pkg/controller/volume/persistentvolume/util"
-	"k8s.io/kubernetes/pkg/controller/volume/scheduling/metrics"
 	"k8s.io/kubernetes/pkg/features"
+	"k8s.io/kubernetes/pkg/scheduler/framework/plugins/volumebinding/metrics"
 	volumeutil "k8s.io/kubernetes/pkg/volume/util"
```

Comment:
Could you re-evaluate which types in this file need to be exported?

I suspect most of them don't need to be anymore. But you can leave that to a follow up PR.

Reply:
Let it be a follow up PR.  There are some inputs:
```
package volumebinding // import "k8s.io/kubernetes/pkg/scheduler/framework/plugins/volumebinding"

CONSTANTS

const ErrReasonBindConflict ConflictReason  - used in k8s.io/kubernetes packages
const ErrReasonNodeConflict ConflictReason  - used in k8s.io/kubernetes packages
const ErrReasonNotEnoughSpace string        - used only in current package (with this patch)
const ErrReasonPVNotExist string            - used only in current package
const DefaultBindTimeoutSeconds string      - Not used in k/k project
)

TYPES

type AssumeCache interface                      - used only in current package
func NewAssumeCache(...) AssumeCache            - used only in current package
type BindingInfo struct {}                      - used only in current package
type CapacityCheck struct {}                    - used only in current package
type ConflictReason string                      - used only in current package
type ConflictReasons []ConflictReason           - used in k8s.io/kubernetes packages
type FakeVolumeBinder struct {}                 - used only in current package
func NewFakeVolumeBinder(...) *FakeVolumeBinder - used in k8s.io/kubernetes packages
type FakeVolumeBinderConfig struct {}           - used in k8s.io/kubernetes packages
type PVAssumeCache interface {}                 - used only in current package
func NewPVAssumeCache(...) PVAssumeCache        - used only in current package
type PVCAssumeCache interface {}                - used only in current package
func NewPVCAssumeCache(...) PVCAssumeCache      - used only in current package
type PodVolumes struct {}                       - used only in current package
type SchedulerVolumeBinder interface {}         - used in k8s.io/kubernetes packages
func NewVolumeBinder(...) SchedulerVolumeBinder - used only in current package
type StorageResource struct {}                  - used only in current package
type VolumeBinding struct {}                    - used in k8s.io/kubernetes packages
type InTreeToCSITranslator interface {}         - used only in current package
```